src/igvfd/schemas/matrix_file.json

```diff
@@ -152,8 +155,10 @@
                     "gene expression",
                     "genomic position",
                     "peak",
+                    "sgRNA",
                     "time",
                     "treatment",
+                    "umi count",
```

Comment:
Technically it should be UMI instead of umi (as UMI = unique molecular identifiers). https://www.illumina.com/techniques/sequencing/ngs-library-prep/multiplexing/unique-molecular-identifiers.html

Reply:
done